ajax/libs/readremaining.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "readremaining.js",
+  "filename": "src/readremaining.jquery.min.js",
+  "version": "1.0.0",
+  "description": "A jQuery plugin that allows you to show the user, how long will take to read the article taking in consideration the speed in which he is scrolling.",
+  "keywords": [
+    "jquery",
+    "ui",
+    "read",
+    "timing"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aerolab/readremaining.js"
+  },
+  "author": "Aerolab",
+  "license": "MIT",
+  "homepage": "http://aerolab.github.io/readremaining.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Aerolab/readremaining.js.git",
+    "basePath": "",
+    "files": [
+      "src/readremaining*",
+      "css/rr*"
```

Comment:
@dannyxx001 can you discuss with the author if it's possible to add a `dist` folder for distribution files?